lib/bike_brigade/delivery/task.ex

```diff
@@ -257,4 +259,10 @@ defmodule BikeBrigade.Delivery.Task do
         changeset
     end
   end
+
+  def fields_for(task) do
```

Comment:
I hate this method name (for `campaign` as well). Anything better? "Normalized fields" or something?